pkgs/top-level/python-packages.nix

```
@@ -8710,6 +8710,8 @@ in {
 
   python3-openid = callPackage ../development/python-modules/python3-openid { };
 
+  python3-otr = callPackage ../development/python-modules/python3-otr { };
```

```suggestion
  otr = callPackage ../development/python-modules/otr { };
```
otr is free and matches better the import name